templates/ilvo/ILVO Source plant.json

```diff
@@ -0,0 +1,148 @@
+      "isa_measurement_type": null,
+      "isa_technology_type": null,
+      "repo_schema_id": "Annotare-default.properties",
+      "organism": "plant",
+      "level": "study"
+    },
+    "data":[
+    {
+      "iri": null,
+      "name": "Source Name",
+      "description": "Sources are considered as the starting biological material used in a study.",
+      "dataType": "String",
+      "title": true,
+      "required": true,
+      "isaTag": "source"
+    },
+    {
+      "iri": "http://purl.obolibrary.org/obo/OBI_0100026",
+      "name": "organism",
+      "description": "A material entity that is an individual living system, such as animal, plant, bacteria or virus, that is capable of replicating or reproducing, growth and maintenance in the right environment. An organism may be unicellular or made up, like humans, of many billions of cells divided into specialized tissues and organs. E.g Drosophila melanogaster.",
+      "dataType": "Ontology",
+      "required": true,
+      "isaTag": "source_characteristic",
+      "ontology": {
+        "name": "efo",
+        "version": "1",
+        "description": "Experimental Factor Ontology",
+        "rootTermURI": "http://purl.obolibrary.org/obo/OBI_0100026"
+      }
+    },
+    {
+        "iri": "http://purl.obolibrary.org/obo/NCBITaxon_33858",
+        "name": "environmental samples",
+        "description": "",
+        "dataType": "String",
+        "required": false,
+        "isaTag": "source_characteristic"
+    },
+    {
+        "iri": null,
+        "name": "remarks",
+        "description": "",
+        "dataType": "String",
+        "required": false,
+        "isaTag": "source_characteristic"
+    },
+    {
+      "iri": "http://www.ebi.ac.uk/efo/EFO_0000399",
+      "name": "developmental stage",
+      "description": "A developmental stage is spatiotemporal region encompassing some part of the life cycle of an organism, e.g. blastula stage",
+      "dataType": "Ontology",
+      "required": true,
+      "isaTag": "source_characteristic",
+      "ontology": {
+        "name": "efo",
+        "version": "1",
+        "description": "Experimental Factor Ontology",
+        "rootTermURI": "http://www.ebi.ac.uk/efo/EFO_0000399"
+      } 
+    },
+    {
+      "iri": "http://www.ebi.ac.uk/efo/EFO_0000246",
+      "name": "age",
+      "description": "A temporal measurement of the time period elapsed since an identifiable point in the life cycle of an organism. If a developmental stage is specified, the identifiable point would be the beginning of that stage. Otherwise the identifiable point must be specified such as planting (e.g. 3 days post planting).",
+      "dataType": "Ontology",
+      "required": true,
+      "isaTag": "source_characteristic",
+      "ontology": {
+        "name": "efo",
+        "version": "1",
+        "description": "Experimental Factor Ontology",
+        "rootTermURI": "http://www.ebi.ac.uk/efo/EFO_0000246"
```

Comment:
Might be best to remove some of the annotare restrictions to input values. The Annotare interface allows for use of non-ontology terms. SEEK enforces the use of the ontology terms, and the ones listed by the attributes don't make sense for plants.

Reply:
I haven't tried lately, but before DataHub also allowed other terms.

Reply:
I believe the test instance is currently not accepting terms that are not part of the chosen Ontology. So it might be best to comment those restrictions out in the Template (for now).